.github/workflows/ci-linux-incremental.yml

```diff
@@ -123,8 +123,7 @@ jobs:
       # Only test systems with a usable system python (>= 3.9)
       # with only a small number of test failures as of 10.2.rc0
       tox_system_factors: >-
-        ["gentoo-python3.11",
-         "archlinux-latest"]
+        ["gentoo-python3.11"]
       tox_packages_factors: >-
```

Comment:
Why deleted "archlinux-latest"?

Reply:
it was failing, and there seems to be nobody who's interested in fixing the "sitepackages" mode on platforms that they don't use themselves

Reply:
"archlinux-latest" is one of the few systems that succeed in CI linux run for 10.3.beta8 https://github.com/sagemath/sage/actions/runs/7894201915 ?

Reply:
Indeed it did. OK, I'll take out this change.

Reply:
Thanks.